x-pack/platform/plugins/shared/maps/public/connected_components/map_container/map_container.tsx

```diff
@@ -227,55 +308,13 @@ export class MapContainer extends Component<Props, State> {
               showTimesliderButton={this.state.showTimesliderButton}
             />
           )}
-          <RightSideControls />
           {this.props.isTimesliderOpen && (
             <Timeslider waitForTimesliceToLoad$={this.props.waitUntilTimeLayersLoad$} />
           )}
+          <RightSideControls />
```

Comment:
This one-line change, moving `<RightSideControls/>` beneath the Timeslider, is the entire fix for the Timeslider focus order. The rest of this PR is related to the flyout.